install/semaphore-install.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: kristocopani
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+    curl \
+    mc \
+    git \
+    gpg \
+    sudo 
+UBUNTU_CODENAME=jammy
```

Comment:
1. Done!
2. Ansible needed for Semaphore else it will fail to setup properly.
3. Jammy needed as per https://docs.ansible.com/ansible/latest/installation_guide/installation_distros.html#installing-ansible-on-debian